catalog.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>NЁRDS</title>
+</head>
+<body>
+	<header>
+		<nav>
+			<a href="index.html">
+				<img src="https://via.placeholder.com/150x100" alt="">
+			</a>
+			<ul>
+				<li><a href="studio.html">Студия</a></li>
+				<li><a href="clients.html">Клиенты</a></li>
+				<li><a href="catalog.html">Магазин</a></li>
+				<li><a href="contacts.html">Контакты</a></li>
+			</ul>
+			<div>
+				<a href="blank.html">Корзина</a>
+			</div>	
+		</nav>
+	</header>
+	<main>
+		<h1>Магазин готовых шаблонов</h1>
+		<sectio>
+			<h2>Фильтр</h2>
+			 <!-- Здесь будет форма фильтра -->
+		</section>
+		<section>
+			<h2>Форма сортировки</h2>
+			 <!-- Здесь будет Форма сортировки: -->
+		</section>
+		<section>
+			<h2>Товары</h2>
+			<ul>
+				<li>
+					<h3>Sedona</h3>
+					<a href="#"><img src="https://via.placeholder.com/150x100"></a>
```

Comment:
Изображению не нужна ссылка. Поэтому просто `img`:
```suggestion
					<img src="https://via.placeholder.com/150x100">
```